.github/workflows/nodejs.yml

```diff
@@ -12,15 +13,39 @@ jobs:
         node-version: [10.x, 12.x]
 
     steps:
+
     - uses: actions/checkout@v1
+
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+
     - name: npm install, build, and test
       run: |
         npm ci
         npm run build
-        npm test
-      env:
-        CI: true
+        npm run test
+
+  coverage:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - name: npm install, build, and test
+      run: |
+        npm ci
+        npm run test-coverage
+
+    - name: Coveralls
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
How has your experience been with github flows, vs. automated hooks directly from the tools (e.g., going to coveralls website and adding the repo)

Reply:
it seems quite straightforward, all I did waws:

- enable rtoken-project access to coveralls
- add the coveralls step to our node CI github action
- go to https://shields.io/category/coverage generate a badge
-done

Reply:
I guess the only difference is that it doesn't show pretty messages here, just ugly ones from github 
![image](https://user-images.githubusercontent.com/35622595/70832403-fdf87180-1dc2-11ea-83ed-c33a03b27adb.png)

Reply:
okay,  yea, I hope github will keep improving!